src/v2/Components/UserSettings/TwoFactorAuthentication/Components/AppSecondFactor/Modal.tsx

```diff
@@ -234,3 +234,34 @@ const InnerForm: React.FC<InnerFormProps> = ({
     </Box>
   )
 }
+
+interface OnCompleteRedirModalProps {
+  onClick: () => void
+  redirectTo: string
+  show: boolean
+}
+
+export const OnCompleteRedirModal: React.FC<OnCompleteRedirModalProps> = props => {
+  const { onClick, redirectTo, show } = props
+
+  return (
+    <Modal
+      title="Set up with app"
+      onClose={onClick}
+      show={show}
+      hideCloseButton={true}
+      FixedButton={
+        <Button onClick={onClick} width="100%">
+          OK
+        </Button>
+      }
+    >
+      <Serif size="3t" color="black60">
+        You’ve successfully set up two-factor authentication!
+      </Serif>
+      <Serif mt={2} size="3t" color="black60">
+        You will be redirected to: {redirectTo}
```

Comment:
url gets wrapped which could be a hassle for users to read. how do we handle long lines in modals?

Reply:
I suggest rendering only the host here (e.g., `cms.artsy.net`). That's more meaningful to users than the `.../auth/artsy` path anyway, and avoids the wrapping concern. (Same for SMS.)